docs/source/basics.rst

```diff
@@ -408,18 +408,62 @@ functions you can use with pyQuil are: ``quil_sin``, ``quil_cos``, ``quil_sqrt``
 Pragmas
 ~~~~~~~
 
+A ``PRAGMA`` directive tells the compiler how it should do its job. Here, we will cover the basics of two very
```

Comment:
Pragmas are more general than that. See [1, sec III.M] and [2, p57, sec 4.3]. I would write something more like:

>`PRAGMA` directives gives users more control over how Quil programs are processed or simulated, but generally do _not_ change the semantics of the Quil program itself. As a general rule of thumb, deleting all `PRAGMA` directives in a Quil program should leave a valid and semantically equivalent program.
>
>In pyQuil, `PRAGMA` directives play many roles, such as controlling the behavior of gates in noisy simulations, or commanding the Quil compiler to perform actions in a certain way. Here, we will cover ...

[1] https://arxiv.org/pdf/1608.03355.pdf

[2] https://www.european-lisp-symposium.org/static/proceedings/2018.pdf